docs/contributing/development.md

```diff
@@ -0,0 +1,198 @@
+
+## What should you work on?
+
+The best way to get going on some actual development is to look through the [issues list](https://github.com/openbullet/OpenBullet2/issues), find an issue you would like to work on, and start hacking! Issues are triaged regularly by the administrative team, and labels assigned that should help you find issues within your skill-set. Once you start working on an issue, please comment on it stating your intent to work on the issue, to avoid unnecessary duplication of work.
+
+### Major Issue Types
+
+A list of issue types can be found on the [issue guidelines](/docs/general/contributing/issues#issue-labels) section.
+
+### What if there isn't an issue?
+
+If there isn't already an issue dealing with the changes you want to make, please [create an issue](/docs/general/contributing/issues) to track it first, then ensure your PR(s) reference the issue in question. This is especially useful for bugs that are found and then fixed by the author, so both the original issue and the fix can be documented and tracked in detail.
+
+## How should you make changes?
+
+Once you've found something you want to work on or improve, the next step is to make your changes in the code, test them, then submit a Pull Request (PR) on GitHub.
+
+For simplicity, all examples assume the developer is operating on Linux with SSH access to GitHub, however the general ideas can be applied to HTTP-based GitHub interfaces, and can be translated to Windows or MacOS.
+
+If you aren't familiar with Git, we recommend the [official documentation](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control) to understand how this version control system works and how to use it.
+
+### Set up your copy of the repo
+
+The first step is to set up a copy of the Git repository of the project you want to contribute to. OpenBullet follows a "fork, feature-branch, and PR" model for contributions.
+
+1. On GitHub, "Fork" the OpenBullet repository you wish to contribute to, to your own user account using the "Fork" button in the relevant repository.
+
+2. Clone your fork to your local machine and enter the directory:
+
+   ```sh
+   git clone git@github.com:yourusername/projectname.git
+   cd projectname/
+   ```
+
+3. Add the "upstream" remote, which allows you to pull down changes from the main project easily:
+
+   ```sh
+   git remote add upstream git@github.com:openbullet2/projectname.git
+   ```
+
+4. To get the `OpenBullet2.Core` project to run successfully, checkout the [openbullet](https://github.com/openbullet/OpenBullet2) project.
+
+5. (TODO)
+
+6. (TODO)
+
+You will now be ready to begin building or modifying the project.
+
+### Make changes to the repo
+
+Once you have your repository, you can get to work.
+
+1. Rebase your local branches against upstream `master` so you are working off the latest changes:
```

Comment:
Code PRs should be opened against the `staging` branch since that one triggers staging pre-releases